lib/main_common.pm

```diff
@@ -1814,8 +1814,8 @@ sub load_extra_tests_filesystem {
     }
     loadtest 'console/snapper_used_space' if (is_sle('15-SP1+') || (is_opensuse && !is_leap('<15.1')));
     loadtest "console/udisks2" unless (is_sle('<=15-SP2') || get_var('VIRSH_VMM_FAMILY') =~ /xen/);
-    loadtest "console/zfs" if (is_leap(">=15.1") && is_x86_64 && !is_jeos);
-    loadtest "network/cifs" unless (is_jeos && is_sle('<15-sp3'));
+    loadtest "console/zfs"  if (is_leap(">=15.1") && is_x86_64 && !is_jeos);
```

Comment:
extra whitespace

Reply:
Result of my tidy script

Reply:
are you sure? :D in this case i would expect the above in the same line. np. you can resolve it if you want. it is just caught my eye and wanted to bring it to your attention

Reply:
Why would I touch `zfs.pm` ? Appears like bottom 2 lines have been aligned...

```
1817     loadtest "console/zfs"  if (is_leap(">=15.1") && is_x86_64 && !is_jeos);
1818     loadtest "network/cifs" if (is_sle('>=15-sp3') || is_opensuse);    
```